### DISCUSSION TITLE
Plase Make image plugin more powerful

### DISCUSSION THREAD

**Discussion by develowper** [codex-team/editor.js#1938] [Ideas]
hi dear developers. Editorjs is great! 
but it will be more powerful with adding some futures :

resize handlers along image
inline-block type for image (can add text beside image)
 upload images on editor save not when image added to block ( u can show preview as blob link and base 64 as file to send)

i saw some of these future requests in issues but there is not any update for now.

Thanks!!!

**Comment by bettysteger** [comment]
Has someone made a resize or an alignment tune?

**Comment by GuilhermeAGouveia** [comment]
for image resize and crop, try this package: https://www.npmjs.com/package/editorjs-image-resize-crop
